code/src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import './header.css';
+
+export const Header = () => {
+	return (
+		<header>
+			<h1>Welcome to the moooooviepage</h1>
+		</header>
+	);
+};
```

Comment:
I can't see this header being displayed on the page?

Reply:
Removed it now, because it added nothing to the page really...